include/deal.II/grid/reference_cell.h

```diff
@@ -1455,6 +1468,62 @@ ReferenceCell::standard_line_to_face_and_line_index(
 }
 
 
+inline unsigned int
+ReferenceCell::line_to_cell_vertices(const unsigned int line,
+                                     const unsigned int vertex) const
+{
+  AssertIndexRange(vertex, 2);
+  AssertIndexRange(line, n_lines());
+
+  if (*this == ReferenceCells::Vertex)
+    return vertex;
+  else if (*this == ReferenceCells::Line)
+    return vertex;
+  else if (*this == ReferenceCells::Quadrilateral)
+    return GeometryInfo<2>::line_to_cell_vertices(line, vertex);
+  else if (*this == ReferenceCells::Hexahedron)
+    return GeometryInfo<3>::line_to_cell_vertices(line, vertex);
```

Comment:
We want to get rid of `GeometryInfo` in favor of `ReferenceCell`. Would you be willing to copy the implementation of these functions here?